tests/python/topi/python/test_topi_sort.py

```diff
@@ -75,7 +75,7 @@ def check_device(device):
         f(tvm_data, tvm_out)
         tvm.testing.assert_allclose(tvm_out.asnumpy(), np_sort, rtol=1e0)
 
-    for device in ["llvm", "cuda", "opencl"]:
+    for device in ["llvm", "cuda", "opencl", "vulkan", "nvptx"]:
```

Comment:
Thanks @masahi  - in the case of vulkan here the tests won't be run until we add vulkan to our CI docker image, right? These are just tested locally?

Reply:
yes vulkan was tested locally, but not on CI.